tests/src/python/test_provider_wfs.py

```diff
@@ -8934,19 +8934,19 @@ def _nam_advanced_request_preprocessor(cls, request: QNetworkRequest, op, conten
                        xmlns:my="http://my"
                        numberMatched="5" numberReturned="0" timeStamp="2016-03-25T14:51:48.998Z">
 </wfs:FeatureCollection>""",
-                b'<?xml version="1.0" encoding="UTF-8"?>\n<wfs:GetFeature xmlns:gml="http://www.opengis.net/gml/3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:wfs="http://www.opengis.net/wfs/2.0" xsi:schemaLocation="http://www.opengis.net/wfs/2.0 http://schemas.opengis.net/wfs/2.0/wfs.xsd" service="WFS" version="2.0.0" xmlns:fes="http://www.opengis.net/fes/2.0" resultType="hits">\n <wfs:Query typeNames="my:typename">\n  <fes:Filter>\n   <fes:Filter xmlns:fes="http://www.opengis.net/fes/2.0">\n    <fes:And>\n     <fes:PropertyIsGreaterThan>\n      <fes:ValueReference>cnt</fes:ValueReference>\n      <fes:Literal>100</fes:Literal>\n     </fes:PropertyIsGreaterThan>\n     <fes:PropertyIsLessThan>\n      <fes:ValueReference>cnt</fes:ValueReference>\n      <fes:Literal>410</fes:Literal>\n     </fes:PropertyIsLessThan>\n    </fes:And>\n   </fes:Filter>\n  </fes:Filter>\n </wfs:Query>\n</wfs:GetFeature>\n': b"""
```

Comment:
Why was the <fes:Filter xmlns ... > removed here?

Reply:
It was incorrectly duplicated